integration/query_comparision_test.go

```diff
@@ -281,9 +281,21 @@ func TestQueryComparisonEq(t *testing.T) {
 				"objectid", "objectid-empty",
 				"regex", "regex-empty",
 				"string", "string-double", "string-empty", "string-whole",
-				"timestamp", "timestamp-i",
+				"testBigFloat64", "testBigInt32", "testBigInt64", "timestamp", "timestamp-i",
 			},
 		},
+		"EqFloat64AndInt64": {
```

Comment:
We don't use `Eq` prefix anymore as this is a part of the test name `TestQueryComparisonEq` itself.

Please move the to the right groups above based on the parameter type. If we rename them as suggested in the other comment, they would fit nicely with other Double, Int32 and Int64 tests

Reply:
Fixed

Reply:
The second part wasn't done. Described it again in the other comment